events/eventServiceInterfaces.go

```diff
@@ -0,0 +1,39 @@
+package events
+
+import (
+	"github.com/FactomProject/factomd/common/constants/runstate"
+	"github.com/FactomProject/factomd/common/interfaces"
+	"github.com/FactomProject/factomd/events/eventinput"
+	"github.com/FactomProject/factomd/events/eventmessages/generated/eventmessages"
+	"github.com/FactomProject/factomd/events/events_config"
+)
+
+type Events interface {
```

Comment:
undescriptive name. The interface is not events, but is a EventEmitter

Reply:
Fixed in  next commit